src/target.rs

```diff
@@ -41,6 +41,7 @@ impl From<String> for Target {
 impl std::fmt::Display for Target {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let target = match self {
+            Target::National => "National ".to_string(),
```

Comment:
is there any reason for this space after `"National"`?

Reply:
not really